### PR TITLE
Grab

### PR DESCRIPTION
**Pull request by oyewoas**

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -15,14 +15,19 @@ const Footer: React.SFC<IFooterProps> = ({}) => {
                     <div className="row">
                         <div className="col-sm-3">
                             <h3 className="footer-heading">Explore</h3>
+                                <li>
+                                    <Link href="/our-story">
```
Comment (IsraelGboluwaga) [inline src/components/Footer/Footer.tsx]: Should't this just be `story`?

File: src/components/contact-form/ContactForm.tsx [hunk 2]
```diff
@@ -19,79 +19,53 @@ interface IProps {
...
                 <div className="container">
                     <div className="row">
-                        <div className="col-sm-4 form-left-side">
+                        <div className="col-sm-5 form-left-side">
                             <h2 className="paragraph-four">
                                 Contact Information
                             </h2>
-                            <p><span><FontAwesomeIcon icon={faMapMarkerAlt}/></span> Gibson House, No 12
-                                Flatten building, Ile Ife.</p>
+                            {/* <p><span><FontAwesomeIcon icon={faMapMarkerAlt}/></span> Gibson House, No 12
```
Comment (IsraelGboluwaga) [inline src/components/contact-form/ContactForm.tsx]: Kindly take out comment

**Review by IsraelGboluwaga — COMMENTED** [review]
Great job, @oyewoas

File: src/components/contact-form/ContactForm.tsx (hunk 2 above)
Comment (oyewoas) [reply src/components/contact-form/ContactForm.tsx]: Done

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (oyewoas) [reply src/components/Footer/Footer.tsx]: Changed it

**Pull request merged**